ios/RNSConvert.h

```diff
@@ -6,6 +6,13 @@ namespace react = facebook::react;
 
 @interface RNSConvert : NSObject
 
++ (NSNumber *)getFontSizePropValue:(int)value;
```

Comment:
Let's leave this particular method in the component implementation file as is not straight up conversion but logic specific to the particular component. 

Alternatively we could name it `NSNumberFromPositiveIntOrNil` or something to make clear what it does.